Comment the "#[doc = !($field)]" line
Upstream has changed this code a lot and this line is not present anymore.

error: attribute value must be a literal
   --> rosenpass/src/msgs.rs:135:21
    |
135 |               #[doc = !($field)]
    |                       ^^^^^^^^^
...
258 | / data_lense! { Envelope<M> :=
259 | |     /// [MsgType] of this message
260 | |     msg_type: 1,
261 | |     /// Reserved for future use
...   |
269 | |     cookie: sodium::MAC_SIZE
270 | | }
    | |_- in this macro invocation
    | 
    = note: this error originates in the macro `data_lense` (in Nightly builds, run with -Z macro-backtrace for more info)


--- rosenpass/src/msgs.rs.orig	2024-05-07 11:12:55 UTC
+++ rosenpass/src/msgs.rs
@@ -132,7 +132,7 @@ macro_rules! data_lense(
         impl<__ContainerType $(, $( $generic: LenseView ),+ )? > $type<__ContainerType $(, $( $generic ),+ )? >{
             $(
             /// Size in bytes of the field `
-            #[doc = !($field)]
+            //#[doc = !($field)]
             /// `
             pub const fn [< $field _len >]() -> usize{
                 $len
